src/molecules/NumberedList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { NumberedListItem } from '../atoms/NumberedListItem';
+
+const NumberedList = ({ items }) => (
+	<React.Fragment>
+		{items.map((item, index) => {
+			return (
+				<NumberedListItem
+					itemProp="recipeInstructions"
```

Comment:
it is `NumberedListItem`, however itemProp is `recipeInstructions` oO

Reply:
recipeInstructions should be possible to be set when NumberedListItem is used as  recipeInstructions. Send it in as props?